cmd/sbctl/enroll-keys.go

```diff
@@ -255,7 +255,7 @@ func KeySync(state *config.State, oems []string) error {
 func RunEnrollKeys(state *config.State) error {
 	// SetupMode is not necessarily required on a partial enrollment
 	ok, err := state.Efivarfs.GetSetupMode()
-	if err != nil {
+	if err != nil && enrollKeysCmdOptions.Export.Value == "" {
```

Comment:
This doesn't look very nice, but the alternative would be to nest the next few lines inside `if enrollKeysCmdOptions.Export.Value == ""  {` which is also not very nice, happy to reformat this if you have a better suggestion.

Reply:
This needs a comment, else I'll totally not get the context later.